packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -323,7 +323,7 @@ export default mixins(
         consoleWarn(`Value must be ${this.isMultiple ? 'an' : 'a'} ${expected}, got ${valueType}`, this)
       }
     },
-    isDateAllowed (value: string) {
+    isDateAllowed (value: string): boolean {
```

Comment:
This type annotation can be removed as others were for consistency, can't it?

Reply:
Same for some ones lower (I think `this.$createElement` has types)

Reply:
Nope, it complains without them. The return type inference seems to be getting worse with each version, you used to be able to have computeds without an explicit return type but now it's needed for methods sometimes too.

Reply:
:disappointed: okay, that's really bad. I only remember broken `this` w/o computed types. Maybe we should open an issue to Vue?